content/includes/header.html

```diff
@@ -68,22 +68,22 @@
 <link rel='stylesheet' href='/assets/css/ie.css'>
 <![endif]-->
 
-<!-- We participate in the US government’s analytics program. See the data at analytics.usa.gov. -->
-<script async type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=VA" id="_fed_an_ua_tag"></script>
-
 <script>
 //<![CDATA[
 window.webpackManifest = 'CHUNK_MANIFEST_PLACEHOLDER';
 //]]>
 </script>
 
-<script src="/generated/vendor.js"></script>
+<script defer src="/generated/vendor.js"></script>
```

Comment:
Do we need to worry about IE <= 9  support for these?

Reply:
No, we support IE11 and up, and display a warning if you use anything older.

Reply:
Our minimum supported version of IE is 11, so no. Worst case scenario, somebody with IE <= 9 opens it up and they load synchronously, right? (I don't actually know that it won't break things, but I think it's a safe assumption.)